src/object/cli_ec.c

```diff
@@ -1101,6 +1153,100 @@ obj_ec_recx_reasb(daos_iod_t *iod, d_sg_list_t *sgl,
 	return rc;
 }
 
+#define obj_ec_set_tgt(tgt_bitmap, idx, start, end)			\
```

Comment:
(style)  Macros with multiple statements should be enclosed in a do - while loop
(style)  macros should not use a trailing semicolon